src/Lykke.Service.ConfirmationCodes/Services/ModelStateValidationActionFilterAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Lykke.Service.ConfirmationCodes.Services
+{
+    public class ModelStateValidationActionFilterAttribute : ActionFilterAttribute
+    {
+        public override void OnActionExecuting(ActionExecutingContext actionContext)
+        {
+            var modelState = actionContext.ModelState;
+
+            if (!modelState.IsValid)
+                actionContext.Result = new BadRequestResult();
```

Comment:
You can insert model state in to the BadRequestResult. It can help to understand which parameter is incorrect